html/field-studies.html

```diff
@@ -0,0 +1,426 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Field Studies Browser</title>
+
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.1/handlebars.min.js"></script>
+    <link rel="stylesheet" href="http://js.arcgis.com/3.14/esri/css/esri.css">
```

Comment:
Line 8 and line 9 need to use //js.arcgis.com instead of http://js.arcgis.com so that we don't have to worry about the protocol.